### PR TITLE
APPCAFE-02: Tableview with mocked data

### PR DESCRIPTION
**Pull request by 1699kratos**
## ISSUE:
* https://trello.com/c/lG01QCS5

## FIX:
* Added a navigation controller
* Added mocked data to the tableView
* Tested in different iphone models
 
## EVIDENCES:
image

image

### CODE REVIEW

File: CafePrices/CafePrices/CafeteriasViewController.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+class CafeteriasViewController: UIViewController {
+    var cafes = [Cafeteria]()
+    @IBOutlet weak var tableView: UITableView!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.delegate = self
+        tableView.dataSource = self
+    }
+
```
Comment (hachiosaki) [inline CafePrices/CafePrices/CafeteriasViewController.swift:18]: delete empty lines

File: CafePrices/CafePrices/TableViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    @IBOutlet weak var cafeteriaName: UILabel!
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        
+    }
+    
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+    }
+
```
Comment (hachiosaki) [inline CafePrices/CafePrices/TableViewCell.swift]: delete empty lines

File: CafePrices/CafePrices/cafeteria.swift [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+//
+//  Created by erick eduardo on 04/10/21.
+//
+
+import Foundation
+
+class Cafeteria {
+    var cafeteriaName: String
+    var cafeteriaImage: String
+    init(cafeteriaName: String, cafeteriaImage: String){
```
Comment (hachiosaki) [inline CafePrices/CafePrices/cafeteria.swift]: add a space after String)

**Review by hachiosaki — CHANGES_REQUESTED** [review]

File: CafePrices/CafePrices/CafeteriasViewController.swift [hunk 4]
```diff
@@ -0,0 +1,36 @@
+//
+//  CafeteriasViewController.swift
+//  Prices
+//
+//  Created by erick eduardo on 04/10/21.
+//
+
```
Comment (hachiosaki) [inline CafePrices/CafePrices/CafeteriasViewController.swift:7]: Remember that the button that leads us to the collection view is no longer needed, please delete it and later on use the didSelectRow method

**Review by hachiosaki — CHANGES_REQUESTED** [review]

File: CafePrices/CafePrices/CafeteriasViewController.swift (hunk 1 above)
Comment (1699kratos) [reply CafePrices/CafePrices/CafeteriasViewController.swift:18]: done

File: CafePrices/CafePrices/TableViewCell.swift (hunk 2 above)
Comment (1699kratos) [reply CafePrices/CafePrices/TableViewCell.swift]: done

File: CafePrices/CafePrices/cafeteria.swift (hunk 3 above)
Comment (1699kratos) [reply CafePrices/CafePrices/cafeteria.swift]: why just at the end and not also at the start ?

File: CafePrices/CafePrices/CafeteriasViewController.swift (hunk 4 above)
Comment (1699kratos) [reply CafePrices/CafePrices/CafeteriasViewController.swift:7]: okey

File: CafePrices/CafePrices/cafeteria.swift (hunk 3 above)
Comment (hachiosaki) [reply CafePrices/CafePrices/cafeteria.swift]: for better understanding

Comment (1699kratos) [reply CafePrices/CafePrices/cafeteria.swift]: okey

File: CafePrices/CafePrices/cafeteria.swift [hunk 5]
```diff
@@ -10,7 +10,7 @@ import Foundation
 class Cafeteria {
     var cafeteriaName: String
     var cafeteriaImage: String
-    init(cafeteriaName: String, cafeteriaImage: String){
+    init(cafeteriaName: String, cafeteriaImage: String ){
```
Comment (hachiosaki) [inline CafePrices/CafePrices/cafeteria.swift]: Sorry if I didn't explain myself, but this was not what I asked for. The space goes between ") {" not between the word and the parenthesis .

**Review by hachiosaki — CHANGES_REQUESTED** [review]
Captura de Pantalla 2021-10-22 a la(s) 21 12 35 
Remove the segue, this was not required.

**Review by hachiosaki — CHANGES_REQUESTED** [review]
Good job

**Pull request merged**